pkg/controller/plan/adapter/vsphere/builder.go

```diff
@@ -1508,6 +1509,38 @@ func (r *Builder) setObjectNameFromTemplate(objectMeta *metav1.ObjectMeta, templ
 	return nil
 }
 
+func (r *Builder) setColdMigrationDefaultPVCName(objectMeta *metav1.ObjectMeta, vm *model.VM, diskIndex int, disk vsphere.Disk) error {
+	pvcNameTemplate := r.getPVCNameTemplate(vm)
```

Comment:
my suggestion is that instead of creating this method we should change the original one to have a default template - so both warm and cold are addressed. in this version of the PR I didn't address warm migration - mostly because I'm not sure how you want to solve it